plugins/pulp_rpm/plugins/importers/yum/parse/rpm.py

```diff
@@ -143,3 +165,137 @@ def package_signature(headers):
         return None
     # gpg program uses the last 8 characters of the fingerprint
     return signature.split()[-1][-8:]
+
+
+def nevra(name):
+    """Parse NEVRA.
+
+    inspired by:
+    https://github.com/rpm-software-management/hawkey/blob/d61bf52871fcc8e41c92921c8cd92abaa4dfaed5/src/util.c#L157. # NOQA
+
```

Comment:
Maybe it is worth to mention why don' t you use hawkey itself but bring some function from it here.